service/frontend/workflowHandler_test.go

```diff
@@ -479,6 +479,23 @@ func (s *workflowHandlerSuite) TestStartWorkflowExecution_Failed_InvalidTaskStar
 	s.Equal(errInvalidTaskStartToCloseTimeoutSeconds, err)
 }
 
+func (s *workflowHandlerSuite) TestRegisterDomain_Failure_MissingDomainDataKey() {
```

Comment:
Could you a test case for checking behavior when dynamic config returns nil? I imagine it will be default until we create a value for it.

Reply:
Other test cases have already cover the case.